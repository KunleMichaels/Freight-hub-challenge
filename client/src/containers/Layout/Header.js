import React from 'react';
// import { connect } from 'react-redux';

const Header = () => {
        return (
            <header className="site-header">
                <nav className="navbar navbar-expand-lg justify-content-start w-100 h-100">
                    <a className="logo-icon" href="/"><img src="https://freighthub.com/wp-content/themes/freighthub/img/logo/logo.svg" alt="logo icon"/></a>
                </nav>
            </header>
        )

}

export default Header;
