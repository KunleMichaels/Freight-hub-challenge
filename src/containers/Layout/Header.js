import React from 'react';
// import { connect } from 'react-redux';

const Header = () => {
        return (
            <header className="site-header">
                <nav className="navbar navbar-expand-lg justify-content-between w-100 h-100">
                </nav>
            </header>
        )

}

export default Header;
