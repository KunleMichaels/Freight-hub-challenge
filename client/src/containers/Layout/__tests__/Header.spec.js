import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Header from "../Header";

Enzyme.configure({ adapter: new Adapter() });

function setup() {
  const enzymeWrapper = shallow(<Header />);
  return {
    enzymeWrapper
  };
}

describe("components", () => {
  describe("Header", () => {
    it("should match snapshot", () => {
      const { enzymeWrapper } = setup();
      expect(enzymeWrapper).toMatchSnapshot();
    });
  });
});
