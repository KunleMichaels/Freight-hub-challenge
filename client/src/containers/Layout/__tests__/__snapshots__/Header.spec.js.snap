// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Header should match snapshot 1`] = `ShallowWrapper {}`;
